### PR TITLE
View Bucket Info

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

File: portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+            <br />
+          </Grid>
+          <Grid item xs={12}>
+            <br />
+          </Grid>
+          <Grid item xs={6}>
+            <Typography variant="h6">Events</Typography>
+          </Grid>
+          <Grid item xs={6} className={classes.actionsTray}>
+            {/*<Button*/}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx:271]: please delete this commented component

File: portal-ui/src/screens/Console/Menu.tsx [hunk 2]
```diff
@@ -116,24 +116,18 @@ class Menu extends React.Component<MenuProps> {
...
             </ListItemIcon>
             <ListItemText primary="policies" />
           </ListItem>
-          <ListItem button component={NavLink} to="/service_accounts">
-            <ListItemIcon>
-              <ServiceAccountIcon />
-            </ListItemIcon>
-            <ListItemText primary="Service Accounts" />
-          </ListItem>
+          {/*<ListItem button component={NavLink} to="/service_accounts">*/}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Menu.tsx:125]: please delete this

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Menu.tsx:125]: done

File: portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx:271]: done

**Pull request merged**